addons/resource/static/src/components/avatar_card_resource/avatar_card_resource_popover.js

```diff
@@ -0,0 +1,61 @@
+/** @odoo-module **/
+
+import { onWillStart } from "@odoo/owl";
+import { useService } from "@web/core/utils/hooks";
+import { useOpenChat } from "@mail/core/web/open_chat_hook";
+import { AvatarCardPopover } from "@mail/discuss/web/avatar_card/avatar_card_popover";
+
+
+export class AvatarCardResourcePopover extends AvatarCardPopover {
+    setup() {
```

Comment:
```suggestion
export class AvatarCardResourcePopover extends AvatarCardPopover {
    static template = "resource.AvatarCardResourcePopover";
    static props = {
        ...super.props,
        recordModel: {
            type: String,
            optional: true,
        },
    };
    static defaultProps = {
        ...super.defaultProps,
        recordModel: "resource.resource",
    };
    setup() {
```

Reply:
> 

Is it not supposed to be equivalent to declare those after the class definition? Do you want me to change that in every js file or is there a reason why this change would be better only in this specific file?

Reply:
Using "static" kw is the way to define static properties. Yes, it should be done in all files :confused: